custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -34,11 +35,11 @@
     // disable otel runtime-metrics instrumentation; we use micrometer metrics instead
     config.put("otel.instrumentation.runtime-metrics.enabled", "false");
 
-    config.put("otel.traces.exporter", "jaeger-thrift-splunk");
+    config.put("otel.traces.exporter", "otlp");
```

Comment:
We should put this into changelog as breaking change.

Reply:
Isn't this the otel default? do we actually need to put this in here at all if we're switching to otlp?

Reply:
We don't, same for propagators: but I thought that being a little explicit won't hurt.

Reply:
I agree with John: no need to explicitly set default values